src/test/java/org/springframework/samples/petclinic/web/PetControllerTests.java

```diff
@@ -171,4 +171,14 @@ void testListAllPetsCorrectInfo() throws Exception {
                 any(Pet.class)
             )));
     }
+    @Test
```

Comment:
Can you put a screenshot of your unit test?
Also, maybe add another unit test? I think that only one test will not be enough